File: old/VS Encrytions cloud/2023/other/PY_Flask/2.py
# -*- coding: utf-8 -*-
# @Author: Dastan_Alam
# @Date:   29-06-2023 01:27:22 PM       13:27:22
# @Last Modified by:   Dastan_Alam
# @Last Modified time: 29-06-2023 02:45:06 PM       14:45:06


from flask import Flask, render_template, send_file
import os
app = Flask(__name__,template_folder="temp")

@app.route('/')
def index():
    return render_template('dowm.html')

@app.route('/download', methods=['GET'])
def download():
    document_path = 'C:\\Users\\harsh\\OneDrive\\MY ENcryption\\VS Encrytions cloud\\2023\\PY_Flask\\upload\\Doc1.docx'
    return send_file(document_path, as_attachment=True)

if __name__ == '__main__':
    app.run(debug=True)

File: VS Encrytions cloud/2022/c c   Encrypt/c   f/2.cpp
/**
 * @Author: Dastan Alam
 * @Date:   08-04-2022  11:55:58 AM  11:04:18
 * @Last Modified by:   DASTAN_ALAM
 * @Last Modified time: 13-05-2022  Fri 12:13:07 PM  12:13:07
 */
#include <iostream>

using namespace std;
// with argument but no return valueee   
void value(int);
int main()
{
    int a=5;
    value(a);
    return 0;
}
void value(int b)
{
    cout<<b;
}